mautic/mautic
# Elfinder enabled sanitizier plugin

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
This plugin enabled sanitizier plugin to remove some characters from filename after upload.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Upload image with empty spaces in file name (`test test.png`)
3. After PR should be sanitize to `test-test.png`
